Examples/Examples/Cart/CartViewController.m

```diff
@@ -94,14 +89,13 @@ - (void)setupSDK {
     [Airwallex setMode:mode];
 
     // Theme customization
-    UIColor *tintColor = [UIColor colorNamed:@"Purple Color"];
-    [AWXTheme sharedTheme].tintColor = tintColor;
+    //    UIColor *tintColor = [UIColor systemPinkColor];
```

Comment:
should we remove these?

Reply:
I thought about removing it, but they're using that `setupSDK` method as a sort of in-code documentation. It is mentioned in the actual documentation, so maybe it's not necessary?

Reply:
Yeah, ideally we should demonstrate how to customise the theme. Is it still possible with the changes?